app/gui/src/project-view/components/visualizations/ScatterplotVisualization.vue

```diff
@@ -576,15 +577,15 @@ watchPostEffect(() =>
     .call(d3.axisLeft(yScale.value).ticks(yTicks.value)),
 )
 
-function getPlotData(data: Data) {
+function getPlotData(data: Data): Point[] {
```

Comment:
optional: you can make the return type `readonly Point[]` to ensure it's not mutated by accident